.github/workflows/build-app.yml

```diff
@@ -204,7 +204,7 @@ jobs:
       - uses: subosito/flutter-action@v2
         name: Set up flutter
         with:
-          flutter-version: '3.24'
+          flutter-version: '3.27.4'
```

Comment:
I think earlier we set this up due to some dependencies breaking up on CI, but its generally good to have it run always against `latest`. @gnunicorn do we want to keep it that way or just having it specified ?

Reply:
this is an attempt to upgrade, if CI agrees, we want to upgrade and CI seems to agree.